src/azure-cli/azure/cli/command_modules/acs/tests/hybrid_2020_09_01/test_aks_commands.py

```diff
@@ -477,7 +477,7 @@ def generate_ppg_id(self, resource_group, location):
     def _get_versions(self, location):
         """Return the previous and current Kubernetes minor release versions, such as ("1.11.6", "1.12.4")."""
         versions = self.cmd(
-            "az aks get-versions -l westus2 --query 'orchestrators[].orchestratorVersion'").get_output_in_json()
+            "az aks get-versions -l westus2 --query 'values[*].patchVersions.keys(@)[]'").get_output_in_json()
```

Comment:
Is this change necessary? To me, the recording file is not changed, so I guess without the change, the test could pass.

Reply:
The unit test that has _get_versions is test_aks_create_service_no_wait which is live_only.

Reply:
@haitch can you confirm?

Reply:
@FumingZhang , I think I just did a search of "get-versions", and update here.

And, I am expecting this need to be updated as well, unless the hybrid test is something different.

Reply:
OK, as the recording file for the hybrid version is not updated, I thought we may keep the test case unchanged. But CI has passed, shouldn't block the change.